common/src/main/java/com/wynntils/commands/ServerCommand.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.commands;
+
+import com.google.common.collect.Lists;
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.arguments.StringArgumentType;
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.tree.LiteralCommandNode;
+import com.wynntils.core.commands.WynntilsCommandBase;
+import com.wynntils.core.webapi.WebManager;
+import com.wynntils.utils.StringUtils;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import net.minecraft.ChatFormatting;
+import net.minecraft.commands.CommandSourceStack;
+import net.minecraft.commands.Commands;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.network.chat.TextComponent;
+
+public class ServerCommand extends WynntilsCommandBase {
+    private final List<String> serverTypes =
+            Lists.newArrayList("WC", "lobby", "GM", "DEV", "WAR", "HB");
```

Comment:
I think this kind of data should be moved to the WC model. I have no specific suggestion, it's just a general note. We want to avoid ending up with the duplication of handling the WC model that plagues the current Wynntils code base. The way to achieve this is to never let actual "features" (which the commands actually are) handle specific knowledge about Wynncraft.